### PR TITLE
Update Changelog for 4.10.0 release

### PR DESCRIPTION
**Pull request by kealjones-wk**
## Motivation
  
4.10.0 needs a changelog update.

## Changes
  
updated changelog.

## Review
_[See CONTRIBUTING.md][contributing-review-types] for more details on review types (+1 / QA +1 / +10) and code review process._

  

Please review: 

### QA Checklist
- [ ] Tests were updated and provide good coverage of the changeset and other affected code
- [ ] Manual testing was performed if needed
    - [ ] Steps from PR author: 
        
    - [ ] Anything falling under manual testing criteria [outlined in CONTRIBUTING.md][contributing-manual-testing]

## Merge Checklist
While we perform many automated checks before auto-merging, some manual checks are needed:
- [ ] A Frontend Frameworks Design member has reviewed these changes
- [ ] There are no unaddressed comments _- this check can be automated if reviewers use the "Request Changes" feature_
- [ ] _For release PRs -_ Version metadata in Rosie comment is correct

[contributing-review-types]: https://github.com/Workiva/over_react/blob/master/CONTRIBUTING.md#review-types
[contributing-manual-testing]: https://github.com/Workiva/over_react/blob/master/CONTRIBUTING.md#manual-testing-criteria

### CODE REVIEW

**Comment by aviary-wf** [comment]
## Security Insights
No security relevant content was detected by automated scans.

## Action Items
* Review PR for [security impact](https://wiki.atl.workiva.net/display/SECURITY/Development+Security+Review+Guidelines); comment "security review required" if needed or unsure
* Verify [`aviary.yaml`](https://wiki.atl.workiva.net/pages/viewpage.action?pageId=42762752#SecurityInsightFramework&Tooling-ConfiguringAviary) coverage of security relevant code
---
_Questions or Comments? Reach out on Slack: #support-infosec._

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,15 @@
 # OverReact Changelog
 
+## [4.10.0](https://github.com/Workiva/over_react/compare/4.9.1...4.10.0)
+- [#829] Add improved prop forwarding method
+- [#809] Update json_annotation & serializable in example app
+- [#826] Allow test_html_builder 3
+- [#827] Raise minimums
```
Comment (greglittlefield-wf) [inline CHANGELOG.md:7]: I don't think these are notable enough to call out individually in the changelog
```suggestion
- [#809], [#826], [#827] Example app dependency updates
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,15 @@
 # OverReact Changelog
 
+## [4.10.0](https://github.com/Workiva/over_react/compare/4.9.1...4.10.0)
+- [#829] Add improved prop forwarding method
```
Comment (greglittlefield-wf) [inline CHANGELOG.md:4]: More specifics on these APIs would be nice so people can find them more easily
```suggestion
- [#829] Add improved prop forwarding methods (`UiProps.getPropsToForward`/`.addPropsToForward`)
```

**Review by greglittlefield-wf — CHANGES_REQUESTED** [review]
Just a couple small things

**Comment by kealjones-wk** [comment]
+1 to @greglittlefield-wf's changes

**Review by greglittlefield-wf — APPROVED** [review]
+10

**Comment by kealjones-wk** [comment]
@Workiva/release-management-p

**Pull request merged**